app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -4,61 +4,92 @@
 import com.codesoom.assignment.domain.Product;
 import com.codesoom.assignment.domain.ProductRepository;
 import com.codesoom.assignment.dto.ProductData;
+import com.github.dozermapper.core.Mapper;
 import org.springframework.stereotype.Service;
 
 import javax.transaction.Transactional;
 import java.util.List;
 
+/**
+ * 상품 도메인의 서비스 레이어.
+ */
 @Service
 @Transactional
 public class ProductService {
+    private final Mapper mapper;
     private final ProductRepository productRepository;
 
-    public ProductService(ProductRepository productRepository) {
+    /**
+     * ProductService 생성자.
+     *
+     * @param productRepository 상품 도메인의 퍼시스턴스 레이어.
+     * @param mapper            객체 매퍼.
+     */
+    public ProductService(ProductRepository productRepository,
+                          Mapper mapper) {
         this.productRepository = productRepository;
+        this.mapper = mapper;
     }
 
+    /**
+     * 상품 목록을 조회한다.
+     *
+     * @return 상품 목록.
+     */
     public List<Product> getProducts() {
         return productRepository.findAll();
     }
 
+    /**
+     * 상품을 조회한다.
+     *
+     * @param id 식별자.
+     * @return 상품.
+     * @throws ProductNotFoundException
```

Comment:
`@throws` 에는 어떤 경우에 던지는지에 대해 작성해 줍니다.
```suggestion
     * @throws ProductNotFoundException 상품을 찾을 수 없는 경우
```